manifests/fastd.pp

```diff
@@ -26,6 +28,15 @@
       ensure => file,
              notify => Service[ffnord::resources::fastd],
              content => template('ffnord/etc/fastd/fastd.conf.erb');
+    "/etc/fastd/${mesh_code}-mesh-vpn/fastd-blacklist.sh":
+      ensure => file,
+             notify => Service[ffnord::resources::fastd],
+             content => template('ffnord/etc/fastd/fastd-blacklist.sh.erb'),
+             mode => '0766';
+   "/etc/fastd/${mesh_code}-mesh-vpn/fastd-blacklist.json":
```

Comment:
@ohrensessel Commit/pull request comments are there to be read, imho. “Adding another option to use the "blacklist feature", i. e. have a list of unwanted peers, anyone else may connect (see https://github.com/ffruhr/fastdbl).” The base idea there is to store them on a central location (github) and update this via cron -- exactly as ffnord-puppet-gateway distributes the fastd-keys across gateways ...
But that's out of scope; this patch just enables that way of controlling fastd use. Without it, ffnord-puppet-gateway can't be used in communities that want to use "fastdbl", as no user would be authenticated (peer directory is empty or not complete).